README.md

```diff
@@ -41,7 +41,7 @@ use Rebilly\Client;
 // Instantiate Rebilly client.
 $client = new Client([
   'apiKey' => ApiKeyProvider::env(),
-  'baseUrl' => Client::SANDBOX_HOST,
+  'base_uri' => Client::SANDBOX_HOST,
```

Comment:
This seems incorrect. The correct property name is `baseUrl`
https://github.com/Rebilly/rebilly-php/blob/6cd1f9be2278fa830c3d8c1c4b70e7e93e0aa59e/src/Client.php#L224

Reply:
Base branch was incorrect. Please ignore.